_layouts/cohort.html

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Could you rename the layout `page-with-toc` or something like this? So we could reuse it not only for cohorts. Thanks

Reply:
> Is there a way to reduce the space between the list elements in the TOC? And maybe also removing the bullets in the list

Yeah. For sure...

I think the bullets make it easier to read though. Don't you think so?

Reply:
Good point. 💯